doc/config.txt

```diff
@@ -19,7 +19,7 @@ List of optional global options::
     toxworkdir=path   # tox working directory, defaults to {toxinidir}/.tox
     setupdir=path     # defaults to {toxinidir}
     distdir=path      # defaults to {toxworkdir}/dist
-    distshare=path    # (DEPRECATED) defaults to {homedir}/.tox/distshare
+    distshare=path    # (DEPRECATED) defaults to $XDG_CACHE_HOME/tox/distshare
```

Comment:
As the docs say: this feature is deprecated, so why should we put extra work into this - or is the deprecation deprecated?